scripts/is/intg/helper-scripts/consolidate_db_is.sh

```
@@ -0,0 +1,247 @@
+
+# Define files and databases
+DB_ENGINE='CF_DBMS_NAME'
+SCRIPT_LOCATION='CF_SCRIPT_LOCATION'
+WSO2_PRODUCT_VERSION_SHORT='CF_PRODUCT_VERSION_SHORT'
+
+if [ $DB_ENGINE = "postgres" ]; then
+  # Create database creation script
+  db_create_file="$WSO2_PRODUCT_VERSION_SHORT/is_postgres_db_create.sql"
+  touch "$db_create_file"
+  echo "-- PostgreSQL Database Creation Script" > "$db_create_file"
+  echo "DROP DATABASE IF EXISTS \"WSO2SHARED_DB\";" >> "$db_create_file"
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)